webpack.mix.js

```diff
@@ -1,6 +1,13 @@
 const mix = require('laravel-mix')
+const path = require('path')
 mix.setPublicPath('./static')
-.setResourceRoot('./resources')
-.disableNotifications(true)
-.ts("./resources/app.tsx","./static/js/main.telescope.js").react()
-.minify("./static/js/main.telescope.js")
+    .setResourceRoot('./resources')
+    .disableNotifications(true)
+    .ts("./resources/app.tsx", "./static/js/main.telescope.js").react()
+    .minify("./static/js/main.telescope.js")
+    .alias({
+        "@app": path.join(__dirname, "resources/app"),
```

Comment:
Codacy has a fix for the issue: [Delete `····`](https://app.codacy.com/gh/bluespada/go-telescope/pullRequest?prid=9653145)
```suggestion
    "@app": path.join(__dirname, "resources/app"),
```